doubly_linked_list/doubly_linked_list.py

```diff
@@ -87,7 +87,14 @@ def remove_from_tail(self):
      return value
 
   def move_to_front(self, node):
-    pass
+    if node == self.head:
+      return None
+
+     self.add_to_head(node.value)
+    if node == self.tail:
```

Comment:
If the node that is moved to the front is not the tail and it is located somewhere in the middle of the DLL, you also need to delete it at its initial position because the previous and next pointers of adjacent nodes when applicable will not be correctly modified.